<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DuongThanhBinh-K244060766</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_bar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bar Chart</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ic_barchart.png</normaloff>Images/ic_barchart.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Chart</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ic_linechart.png</normaloff>Images/ic_linechart.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pie Chart</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ic_piechart.png</normaloff>Images/ic_piechart.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Images/ic_shutdown.png</normaloff>Images/ic_shutdown.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Chart Visualization</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
